### ISSUE TITLE
supercollider: Another instance of "Could not load the Qt platform plugin"

### ISSUE THREAD

**Issue by turion** [NixOS/nixpkgs#116548]
Starting supercollider results in the infamous "Could not load the Qt platform plugin" bug.

Reproduce:

1. `nix-shell -p supercollider`
2. `sclang`

```
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb, wayland-org.kde.kwin.qpa.
```

I tried it as well on nixpkgs master in a clean nixos test VM.

**Notify maintainers**
@MrMebelMan 

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.77, NixOS, 21.03pre252083.2deeb58f494 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.8`
 - channels(root): `"musnix, nixos-21.03pre268206.536fe36e23a, nixos-hardware"`
 - channels(turion): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - supercollider
# a list of nixos modules affected by the problem
module:
```

**Comment by turion** [comment]
CC @ttuegel I believe?

**Comment by ttuegel** [comment]
Please refer to the Nixpkgs manual [section](https://nixos.org/manual/nixpkgs/stable/#sec-language-qt) about packaging Qt applications, particularly the section labeled "Locating runtime dependencies" regarding `wrapQtAppsHook`.

**Comment by OPNA2608** [comment]
Should be fixed by #117333, closing.
